platform/ios/test/MapViewTests.m

```diff
@@ -424,6 +424,29 @@ - (void)testInsetMapView {
                   @"compass should lie inside shrunken map view");
 }
 
+- (void)testContentInsetsWithTinyMapView {
+    [tester.viewController tinyMapView];
+    [self keyValueObservingExpectationForObject:tester.mapView keyPath:@"contentInset" handler:^BOOL(id observedObject, NSDictionary *change) {
+        XCTAssertEqual(tester.mapView.contentInset.top,
+                       0,
+                       @"map should not have top content inset");
+        XCTAssertEqual(tester.mapView.contentInset.bottom,
+                       0,
+                       @"map should not have bottom content inset");
+        return YES;
+    }];
+    [self waitForExpectationsWithTimeout:5.0 handler:nil];
```

Comment:
Five seconds seems rather generous. Is it necessary to make this test take 10 seconds for a frame change that should be instant?